Message.java

```diff
@@ -0,0 +1,20 @@
+
+public class Message {
+	
+	private String toAddress;
+	private String fromAddress;
+	private String message;
+	
+	public void getTo(String toAddress) {
```

Comment:
http://stackoverflow.com/questions/6638964/set-and-get-methods-in-java

Data protection can get confusing sometimes. I'll create a project  and push to master tonight with your classes added.